boards/nxp/imx95_evk/imx95_evk_mimx9596_m7.dts

```diff
@@ -23,6 +23,24 @@
 	};
 };
 
+&emdio {
+	pinctrl-0 = <&emdio_default>;
+	pinctrl-names = "default";
```

Comment:
I know these nodes are disabled in soc dts, but I prefer to put explict "disabled" status for these nodes to make it more readable for the user to know it is disabled currently.

Reply:
okay, updated.
Thanks.